onboarding_papers/FishSense.bib

```diff
@@ -0,0 +1,9 @@
+@INPROCEEDINGS{tueller_maddukuri_paxson_et_al_oceans_2021,
```

Comment:
I think we should add the underwater calibration paper.

Reply:
Fixed in 1559885251d7175f08bf7ba9d159c33efc95574a